nixos/tests/ctrld.nix

```diff
@@ -0,0 +1,25 @@
+
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "ctrld";
+
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ arthsmn ];
+  };
+
+  nodes = {
+    # machine = { pkgs, ... }: {
+    #   environment.systemPackages = [ pkgs.dig ];
+    # };
```

Comment:
Were there any plans for this, or should it be removed?

Reply:
Sorry, I was testing and forgot to uncomment. Done.